ansible/roles/ssh/tasks/main.yml

```diff
@@ -17,9 +17,9 @@
 - name: Ensure challenge response access is disallowed
   lineinfile:
     dest: /etc/ssh/sshd_config
-    regex: "^ChallengeResponseAuthentication"
+    regexp: "^ChallengeResponseAuthentication"
     line: "ChallengeResponseAuthentication no"
-    state: absent
+    state: present
```

Comment:
changed from removing `yes` line to adding explicit `no` line, and forgot to update